lab_5_scrapper/scrapper.py

```diff
@@ -204,12 +371,28 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    base_path.mkdir(parents=True, exist_ok=True)
+    for file in base_path.iterdir():
+        file.unlink(missing_ok=True)
```

Comment:
Consider removing the whole directory and then creating it from scratch rather than deleting each file individually.